provider.go

```diff
@@ -91,13 +94,37 @@ func (c *Client) Provider(streamID StreamID, tracks []TrackEntry) (*Provider, er
 }
 
 type BlockWriter interface {
+	// Write a block to Kinesis Video Stream.
 	Write(*BlockWithBaseTimecode) error
+	// Read a response from Kinesis Video Stream.
```

Comment:
Follow the godoc style.
```suggestion
	// ReadResponse reads a response from Kinesis Video Stream.
```